content/docs/studio/user-guide/projects-and-experiments/visualize-and-compare.md

```diff
@@ -31,37 +30,48 @@ Iterative Studio can work with two types of plots files in your repository:
 2. Image files in JPEG, GIF, PNG, or SVG format. These images will be displayed
    directly in Iterative Studio.
 
-You can define multiple plots in a single repository. Below is an example
-snippet from a `dvc.yaml` file showing the `evaluate` stage of the DVC pipeline.
+You can define multiple plots in a single repository. The plots can be
+[stage-specific](/doc/user-guide/experiment-management/visualizing-plots#plot-outputs)
+or
+[top-level](/doc/user-guide/experiment-management/visualizing-plots#top-level-plots).
+Below is an example snippet from a `dvc.yaml` file showing the `evaluate` stage
+of the DVC pipeline, with 2 different plots. It also includes a top-level plot
+which uses data from the file `runtime_logs/logs.csv`.
 
 ```yaml
-evaluate:
-  cmd: python src/evaluate.py
-  deps:
-    - output/data.pkl
-    - output/model.h5
-    - src/evaluate.py
-  metrics:
-    - output/metrics.json:
-        cache: false
-  plots:
-    - output/predictions.json:
-        cache: false
-        template: confusion
-        x: actual
-        y: predicted
-    - output/misclassified_samples/:
-        cache: false
+stages:
+  evaluate:
+    cmd: python src/evaluate.py
+    deps:
+      - output/data.pkl
+      - output/model.h5
+      - src/evaluate.py
+    metrics:
+      - output/metrics.json:
+          cache: false
+    plots:
+      - output/predictions.json:
+          cache: false
+          template: confusion
+          x: actual
+          y: predicted
+      - output/misclassified_samples/:
+          cache: false
+  ...
+  ...
+plots: runtime_logs/logs.csv
```

Comment:
```suggestion
  evaluate:
    cmd: python src/evaluate.py
    deps: ...
    plots:
      - output/predictions.json: 
          template: confusion
          x: actual
          y: predicted

plots: output/misclassified_samples/ 
```

☝🏼 The top-level example is somewhat plain though, not really showing the benefit of flexible plots.

Reply:
@jorgeorpinel I've incorporated the changes, but with some modification. PTAL. Specifically, I didn't move the image plots to top-level, because `misclassified_samples` are usually the output of model evaluation.

Reply:
OK